src/processTokens.ts

```diff
@@ -132,8 +133,14 @@ export async function processTokens(
                 ...tokenConfig.chains[chainId],
             };
             const tokenData = {name, symbol, address, decimals, chainId};
-            if (logoURI) {
-                Object.assign(tokenData, {logoURI});
+            if (await doesLogoExist(primaryAddress, chainId)) {
+                Object.assign(tokenData, { 
+                    logoURI: `https://buttonwood-protocol.github.io/buttonwood-token-list/assets/tokens/${chainId}/${primaryAddress}.png`,
+                });
+            } else {
```

Comment:
This breaks the checking that asset actually exists (and is why logoURIs have been added for a bunch of assets in the output tokenlist, despite them failing to load)

I think making line 140 an `else if(await doesLogoExist(primaryAddress){` would resolve this

Reply:
I think we should aim to keep logoURIs that don't have an asset where they're pointing to out of the tokenlist

Reply:
cool sounds good

Reply:
fixed